package/base-files/files/bin/ipcalc.sh

```diff
@@ -90,8 +90,7 @@ BEGIN {
 	}
 
 	if (ipaddr > start && ipaddr < end) {
-		print "ipaddr inside range" > "/dev/stderr"
-		exit(1)
+		print "warning: ipaddr inside range - this might not be supported in future releases of Openwrt" > "/dev/stderr"
```

Comment:
I would leave the `exit(1)` but comment it out.  In the future, when users have had a chance to correct their configurations, then we can start enforcing it with the `exit`.

Reply:
please resolve if you're happy with the change